c++/src/kj/common.c++

```diff
@@ -44,6 +44,15 @@ void unreachable() {
   KJ_KNOWN_UNREACHABLE(abort());
 }
 
+thread_local bool currentThreadId;
```

Comment:
Declare this `static` and/or put it in an anonymous namespace to make sure the compiler gives it internal linkage.

(I think `static` is assumed for variables unless you explicitly declare them `extern` but doesn't hurt to be explicit.)